_schemaV2.graphql

```diff
@@ -7114,6 +7115,10 @@ type Mutation {
   ): UpdateViewingRoomSubsectionsPayload
 }
 
+type MyCollectionArtworkMutationDeleteSuccess {
```

Comment:
I'm assuming this was just missed in #2654, and doesn't actually have anything to do with the spirit of this PR, right?

Reply:
Hmm not too sure why these changes are here, but they look right!